src/test/java/scenarios/TransactionTest.java

```diff
@@ -762,7 +763,7 @@ public void cverifyThatElementsOfTransactionDetailsPopup() {
         session.getTransactionsPage().selectStore(StoreAccount.AutomationTransactions3);
         TransactionsPage transaction = session.getTransactionsPage();
         transaction.getCurrentPaidBill().clickByMouse();
-        WebdriverWaits.sleep(3000);
+        WebdriverWaits.waitForElementVisible(transaction.transactionID,3);
 
         // Verifying the elements on Transaction Popup
         Assertions.assertTrue(transaction.getTransactionID().isDisplayed());
```

Comment:
In line 369 likewise, Can combine multiple assertions into a single assertion block for better error reporting and efficient test execution.